src/cli.rs

```diff
@@ -66,6 +74,7 @@ pub fn args() -> Args {
         all: all,
         subcommand: sc,
         target: target,
-        message_format: message_format
+        message_format: message_format,
+        root: root,
```

Comment:
```suggestion
        root,
```
Feel free to also do this for the other fields; this code likely predates that Rust feature.^^

Reply:
fixed